src/platform/STM32/mk/STM32F7.mk

```diff
@@ -185,5 +186,8 @@ MSC_SRC = \
             msc/usbd_storage_sdio.c \
             msc/usbd_storage_sd_spi.c
 
+SPEED_OPTIMISED_SRC := $(SPEED_OPTIMISED_SRC) \
```

Comment:
maybe `+=` ?

Reply:
Yeah I copied as is, but I agree += is cleaner.